source/codegen/metadata/nidcpower/functions_addon.py

```diff
@@ -0,0 +1,639 @@
+# These dictionaries are merged with the extracted function metadata at build time.
+# Changes to the metadata should be made here, because functions.py is generated thus any changes get overwritten.
+
+functions_override_metadata = {
+}
+
+functions_additional_create_advanced_sequence = {
```

Comment:
I think you can just delete this, I don't believe it's used or intended to be used in through the grpc server.

Reply:
@reckenro, Done deleting.